scripts/install-mender.sh

```diff
@@ -88,8 +93,14 @@ is_known_component() {
 parse_args() {
     local selected_components=""
     local args_copy="$@"
-    while [ $# -gt 0 ]
-    do
+    local passthrough=0
+    MENDER_SETUP_ARGS=""
+    while [ $# -gt 0 ]; do
+        if test $passthrough -eq 1; then
+            MENDER_SETUP_ARGS="$MENDER_SETUP_ARGS $1"
```

Comment:
Question:
Once `passthrough` is 1, can it ever not be?
If so, then it's not really needed to loop in order to collect the remaining arguments right?

Reply:
Yes, you are right. We can terminate the processing at once after `--` and assign the remaining arguments to `MENDER_SETUP_ARGS` :+1: